browser/js/bundle.entry.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  bel: require("bel"),
+  morphdom: require('morphdom'),
```

Comment:
You could use `yo-yo` here instead of `bel` and `morphdom` separately. It has default behavior around copying events which you might want.